client/ayon_core/plugins/publish/collect_otio_review.py

```diff
@@ -36,20 +36,29 @@ def process(self, instance):
         # optionally get `reviewTrack`
         review_track_name = instance.data.get("reviewTrack")
 
+        # [clip_media] setting:
+        # Extract current clip source range as reviewable.
+        # Flag review content from otio_clip.
+        if not review_track_name and "review" in instance.data["families"]:
+            otio_review_clips = [otio_clip]
+
+        # skip if no review track available
+        elif not review_track_name:
+            return
+
         # generate range in parent
         otio_tl_range = otio_clip.range_in_parent()
 
         # calculate real timeline end needed for the clip
         clip_frame_end = int(
             otio_tl_range.start_time.value + otio_tl_range.duration.value)
 
-        # skip if no review track available
-        if not review_track_name:
-            return
-
         # loop all tracks and match with name in `reviewTrack`
         for track in otio_timeline.tracks:
-            if review_track_name != track.name:
+            if (
+                review_track_name is None
+                or review_track_name != track.name
+            ):
```

Comment:
I think you can keep it as it was, or at least break the loop early.
```suggestion
            if review_track_name != track.name:
```

```suggestion
            # Skip the loop
            if review_track_name is None:
                break

            if review_track_name != track.name:
```